File: back/src/main/java/csh/back/domain/member/dto/response/AuthFilterDto.java
package csh.back.domain.member.dto.response;

import csh.back.domain.member.entity.Member;

public record AuthFilterDto (
		Long id,
		String email
) {
	public static AuthFilterDto from(Member member) {
		return new AuthFilterDto(
				member.getId(),
				member.getEmail()
		);
	}
}

File: back/src/main/kotlin/csh/back/domain/member/dto/response/AuthFilterDto.kt
package csh.back.domain.member.dto.response

import csh.back.domain.member.entity.Member

data class AuthFilterDto(
    val id: Long,
    val email: String,
) {
    companion object {
        fun from(member: Member) = AuthFilterDto(
            id = member.id!!,
            email = member.email,
        )
    }
}